vaadin-card-flow-parent/vaadin-card-flow/src/test/java/com/vaadin/flow/component/card/CardTest.java

```diff
@@ -68,6 +68,39 @@ public void setTitle_slotAttributeSet() {
         setSlotContent_slotAttributeIsSet(Card::setTitle, "title");
     }
 
+    @Test
+    public void stringTitleIsEmptyByDefault() {
+        Assert.assertEquals("", card.getTitleAsText());
+    }
+
+    @Test
+    public void setStringTitle_titleIsSet() {
```

Comment:
Let's also add assertions that the respective element properties are set.

Reply:
Added the assertions.